hubmap/dataportal/decorators.py

```diff
@@ -0,0 +1,32 @@
+from functools import wraps
+
+from django.contrib.auth.models import *
+from django.core.exceptions import *
+from django.core.exceptions import PermissionDenied
+
+from . import views
+from .models import *
+
+ALLOWED_GROUPS = [
+    'dataportal_admin',
+    'dataportal_db_admin',
+]
+
+def user_has_view_permissions(function):
```

Comment:
This is currently unused, so it should probably have a docstring noting its purpose: example code for group-based access control for the API.